fuse_optimizers/src/batch_optimizer.cpp

```diff
@@ -289,19 +294,19 @@ void BatchOptimizer::start()
 void BatchOptimizer::stop()
 {
   ROS_INFO_STREAM("Stopping optimizer.");
-  // Tell all the plugins to stop
-  stopPlugins();
+  started_ = false;
```

Comment:
Actually, this 'started_' variable itself is a shared resource as well between main thread and the optimization thread. But it only changes in main thread. So no protection should be okay.

Reply:
In this case, started is an atomic bool, so we are safe.